site/content/user-guide/amqp-adapter.md

```diff
@@ -0,0 +1,175 @@
+
+### SASL PLAIN Authentication
+
+The AMQP adapter currently supports authenticating clients using a *username* and *password*. This means that clients needs to provide a *username* and a *password* when connecting to the AMQP adapter. The *username* must match the pattern [*auth-id@tenant*], e.g. `sensor1@DEFAULT_TENANT`. The adapter verifies the credentials provided by the client against the credentials that the [Credentials Service] ({{< relref "#credentials-service-connection-configuration" >}}) has on record for the device. If the credentials match, then authentication is successful and the client device can proceed to publish messages to Hono.
+
+When running the Hono example installation, as described in the [Getting Started guide]({{< relref "getting-started.md" >}}), the demo Credentials service comes pre-configured with a `hashed-password` secret for devices `4711` and `gw-1` of tenant `DEFAULT_TENANT` as shown below. These credentials are used in the following examples to illustrate the usage of the adapter. We use the **qpid-send** command-line tool (from the Apache Qpid project) to simulate a device publishing messages to the AMQP adapter.
+
+{{% note %}}
+There is a subtle difference between the *device identifier* (*device-id*) and the *auth-id* a device uses for authentication. See [Device Identity]({{< relref "concepts/device-identity.md" >}}) for a discussion of the concepts.
+{{% /note %}}
+
+### SASL EXTERNAL Authentication
+TODO
+
+## Publish Telemetry Data (authenticated Device)
+
+The AMQP adapter supports publishing of telemetry to Hono's Telemetry API. Telemetry messages are published using both `AT_LEAST_ONCE` and `AT_MOST_ONCE` delivery semantics.
+
+* Address: `telemetry`
+* Delivery: `AT_MOST_ONCE` or `AT_LEAST_ONCE`
+* Authentication: required
+* Message Header(s):
+  * (optional) Arbitrary headers (durable, ttl, priority, ...)
+* Message Body:
+  * (optional) Arbitrary payload
+* Message properties:
+  * (optional) Arbitrary properties (content-type, correlation-id, ...)
+* Disposition Frames:
+  * Accepted: Message successfully processed by the adapter.
+  * Released: Message cannot be processed and should be redelivered.
+  * Rejected: Adapter rejects the message due to one of the following:
+        * (`hono:bad-request`): Request rejected due to a bad client request.
+        * (`amqp:unauthorized-access`): Request rejected because the adapter is disabled for tenant.
+        * (`amqp:precondition-failed`): Request does not fulfill certain requirements e.g adapter cannot assert device registration etc.
+
+When a device publishes data to the `telemetry` address, the AMQP adapter automatically determines the device's identity and tenant during the authentication process.
+
+**Example**
+
+Publish some JSON data for device `4711`:
+
+    $ qpid-send -b "localhost:4040" --connection-options "{username:sensor1@DEFAULT_TENANT, password:hono-secret, protocol:amqp1.0}" -a "'telemetry'" --content-string '{"temp": 5}'
```

Comment:
I am quite sure that not many people will be familiar with `qpid-send`, you should at least include a link to the Apache Qpid project page to point them in the right direction